src/configure

```diff
@@ -212,13 +212,17 @@ function linux_configure_mkl_libraries {
       local libfile=$mkllibdir/lib$file.so
       check_exists $libfile
     done 
-    echo  "-L$mkllibdir -Wl,-rpath=$mkllibdir -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm"
+    #echo  "-L$mkllibdir -Wl,-rpath=$mkllibdir -lmkl_intel_lp64 -lmkl_intel_thread -lmkl_core -liomp5 -lpthread -lm"
+    linkline="-L$mkllibdir -Wl,-rpath=$mkllibdir  -lmkl_intel_lp64 -lmkl_core -lmkl_tbb_thread -ltbb -lstdc++ -lpthr$
```

Comment:
Looks like the line was cut and the end mangled. It fails but way below because of the quoting: https://travis-ci.org/kaldi-asr/kaldi#L1370